### DISCUSSION TITLE
GPodder API Wiki wrong info

### DISCUSSION THREAD

**Discussion by ghost** [SamTV12345/PodFetch#245] [General]
[The wiki](https://github.com/SamTV12345/PodFetch/wiki/GPodder-API) says:

> If you have PodFetch running in Docker just execute:
>
> `docker exec -it podfetch /app/podfetch users add`

This is incorrect if the user's container name is not `podfetch`

Should be updated to:

```
docker ps #This will help you obtain the container's id and name
docker exec -it <container id or name> /app/podfetch users add
```

Can also mention about invites too:

> You can also generate a user using the invite function within the GUI
>
> To generate an invite, log into Podfetch → Top Right Icon → User Administration → Invites

**Comment by SamTV12345** [comment]
Thanks for the hint with the documentation issue. I included your feedback. Thanks for taking a look at the wiki.
